src/Javis.jl

```diff
@@ -491,28 +500,49 @@ function perform_transformation(trans::InternalRotation)
     rotate(trans.angle)
 end
 
-pos(p::Point) = p
-pos(t::Transformation) = t.p
+"""
+    val(s::Symbol)
+
+Get access to the value that got saved in `s` by a previous action.
+If you want to access a position or angle check out [`pos`](@ref) and [`angle`](@ref).
```

Comment:
Similarly to what we discussed with `ang` and `pos` in #66, can you also give an alternative name like `get_value`? Thanks!

Reply:
I'll do this in #66 and merge that one first then.